types/api/apps.d.ts

```diff
@@ -23,15 +23,15 @@ type AppsState = {
 };
 
 type AppBinding = {
-    app_id: string;
-    location: string;
+    app_id?: string;
+    location?: string;
```

Comment:
Should we be more strict and require a `location`?

Reply:
Same as in the previous comment.